pyquil/tests/test_reference_density_simulator.py

```diff
@@ -264,6 +264,26 @@ def test_multiqubit_decay_bellstate():
     assert np.allclose(qam.wf_simulator.density, state)
 
 
+# make an abstract compiler
```

Comment:
This is also duplicated in `test_quantum_computer.py`. Both could be moved into `utils.py` and then imported when needed.

Reply:
- [x] todo